### ISSUE TITLE
 `clickComponent` lookupFactory of undefined error

### ISSUE THREAD

**Issue by alexdiliberto** [201-created/ember-cli-acceptance-test-helpers#30]
Anytime I call `clickComponent`, I get the following error. I even created a dummy app locally in its most pared down form to reproduce.

```bash
TypeError: Cannot read property 'lookupFactory' of undefined
    at lookupComponent (lookup.js:11)
    at clickComponent (async.js:6)
    at ember.debug.js:40913
    at isolate (ember.debug.js:41114)
    at ember.debug.js:41098
    at tryCatch (ember.debug.js:53543)
    at Object.invokeCallback (ember.debug.js:53558)
    at ember.debug.js:55684
    at ember.debug.js:32082
    at Queue.invoke (ember.debug.js:333)
```

The issue stems from the following commit https://github.com/201-created/ember-cli-acceptance-test-helpers/commit/2cfc1b84174baf9f5aa75b92e177772af1cd013d

Where now in v0.4.0+ `app.container` will be undefined https://github.com/201-created/ember-cli-acceptance-test-helpers/blob/v0.4.0/test-support/helpers/201-created/async.js#L5

I believe something like the following which you're already doing would fix the issue: https://github.com/201-created/ember-cli-acceptance-test-helpers/blob/v0.4.0/test-support/helpers/201-created/raw/expect-component.js#L8-L12

**Comment by alexdiliberto** [comment]
I would be happy to submit a PR and failing test if needed

**Comment by alexdiliberto** [comment]
friendly ping @iezer

**Comment by iezer** [comment]
@alexdiliberto Thanks for raising this. Can you please open a PR and we'll get it merged?

**Comment by alexdiliberto** [comment]
@iezer Forgot to link back here, It completed that PR

https://github.com/201-created/ember-cli-acceptance-test-helpers/pull/31

**Comment by iezer** [comment]
@alexdiliberto Thanks again for your help. I've released this as [0.4.2](https://www.npmjs.com/package/ember-cli-acceptance-test-helpers) I'm closing this issue. Please let us know if you have any other feedback.
